include/contraction/contract.hpp

```diff
@@ -54,55 +55,80 @@ class Pgr_contract {
         G &graph,
         Identifiers<V> forbidden_vertices,
         std::vector<int64_t> contraction_methods,
-        int64_t max_cycles
-    ) {
-        std::deque<int64_t> contract_order;
+        int64_t max_cycles,
+        std::ostringstream &log
+    ) 
+    {
+        std::deque<int64_t> contraction_methods_deque;
```

Comment:
`contraction_sequence_deque`? (In echo to my comment in #6.)

Reply:
Changed to `contract_order`